que.gemspec

```diff
@@ -27,8 +27,8 @@ Gem::Specification.new do |spec|
   # This is highly non ideal, but unless we properly fork, we have to do this for now.
   spec.add_dependency "prometheus-client"
 
-  spec.add_dependency "rack", "~> 2.0"
-  spec.add_dependency "webrick", "~> 1.7"
+  spec.add_dependency "rack", "~> 3.0"
```

Comment:
Any major changes to look out for? I don't actually know if the Prom exporters are integration tested

Reply:
https://github.com/rack/rack/blob/main/UPGRADE-GUIDE.md

Yeah some stuff, main ones seems to moving out rackup into its own gem and that the response array is now expected to be mutable

Reply:
I'm pretty sure that all the gems that would be affected have already been updated to either use Rack 3 or to be compatible with both 2 and 3 because the upgrade to Rails 7.1 made it necessary